FiskalySDKTests/FiskalyAPITests.swift

```diff
@@ -16,6 +16,7 @@ class FiskalyAPITests: XCTestCase {
     }
 
     func testManagementRequest() throws {
+        do {
```

Comment:
Doesn't the `throws` keyboard in the test case signature already do the same? Usually, the test report contains similar information.

Reply:
I think I put this in when the test was failing so that I could look at the error in the debugger, but now that the error has a debugDescription there's more useful information in the test report, so this is not necessary any more. I'll remove it. :)